Mu2eG4/src/constructDiskCalorimeter.cc

```diff
@@ -253,7 +253,14 @@ namespace mu2e {
        G4double pipeThickness            = cal.caloInfo().getDouble("pipeThickness");
        G4double pipeInitSeparation       = cal.caloInfo().getDouble("pipeInitSeparation");    
        std::vector<double> pipeTorRadius = cal.caloInfo().getVDouble("pipeTorRadius");
-              
+       std::vector<double> LargeTorPhi    = cal.caloInfo().getVDouble("LargeTorPhi");
+       std::vector<double> smallTorPhi   = cal.caloInfo().getVDouble("smallTorPhi");
+       std::vector<double> yposition     = cal.caloInfo().getVDouble("yposition");
+       std::vector<double> straitEndPhi  = cal.caloInfo().getVDouble("straitEndPhi");
+       G4double radSmTor               = cal.caloInfo().getDouble("radSmTor");
```

Comment:
Somehow, the indentation is all wrong in this file

Reply:
looks fine to me, what is wrong?

Reply:
The issue is that the file is full of hard tabs.  Leave it for now.  We will clean up the whole repo soon-ish.

For future reference, there is a lot of needless copying going on here.  But it requires changes to DiskCalorimeter.  Nothing to do on this PR.